common/static/common/js/discussion/utils.js

```diff
@@ -184,10 +184,8 @@
             if (!params.error) {
                 params.error = function() {
                     self.discussionAlert(
-                        gettext('Sorry'),
-                        gettext(
-                            'We had some trouble processing your request. Please ensure you have copied any ' +
-                            'unsaved work and then reload the page.')
+                        gettext('Error'),
```

Comment:
@alisan617 Is this what you were referring to, re comma instead of period after what used to be "Sorry"? I had intended the entire string "Sorry" to be removed, did not realize it would be replaced by "Error". 
@andy-armstrong I haven't able to create this error in the sandbox, but I think we could/should delete ```gettext('Error'),``` altogether. (All occurrences, not just this line).